scenario29/jmeter/Solution29.jmx

```diff
@@ -19,81 +19,101 @@
       <hashTree/>
       <Arguments guiclass="ArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
         <collectionProp name="Arguments.arguments">
-          <elementProp name="permission" elementType="Argument">
-            <stringProp name="Argument.name">permission</stringProp>
-            <stringProp name="Argument.value">${__property(permission)}</stringProp>
-            <stringProp name="Argument.desc">Permission for role</stringProp>
+          <elementProp name="isHost" elementType="Argument">
+            <stringProp name="Argument.name">isHost</stringProp>
+            <stringProp name="Argument.value">is540B.qa.com</stringProp>
             <stringProp name="Argument.metadata">=</stringProp>
           </elementProp>
-          <elementProp name="claimURI" elementType="Argument">
-            <stringProp name="Argument.name">claimURI</stringProp>
-            <stringProp name="Argument.value">${__property(claimURI)}</stringProp>
+          <elementProp name="isPort" elementType="Argument">
+            <stringProp name="Argument.name">isPort</stringProp>
+            <stringProp name="Argument.value">443</stringProp>
             <stringProp name="Argument.metadata">=</stringProp>
           </elementProp>
-          <elementProp name="claimValue" elementType="Argument">
-            <stringProp name="Argument.name">claimValue</stringProp>
-            <stringProp name="Argument.value">${__property(claimValue)}</stringProp>
+          <elementProp name="host" elementType="Argument">
+            <stringProp name="Argument.name">host</stringProp>
+            <stringProp name="Argument.value">is540B.qa.com</stringProp>
```

Comment:
Hardcoded server name is540B.qa.com please correct

Reply:
Hi @chaminda,

This was fixed by 2nd commit [1] in this pull request.

[1] https://github.com/wso2-incubator/identity-test-integration/pull/165/commits/047f3ba11ddddca4c3878134ad7114c2cc6811f7